build.gradle.kts

```diff
@@ -40,7 +41,14 @@ dependencies {
 	testImplementation("io.kotest:kotest-assertions-core-jvm:5.0.0")
 	testImplementation("io.mockk:mockk:1.13.12")
 	testRuntimeOnly("org.junit.platform:junit-platform-launcher")
-	testImplementation("io.mockk:mockk:1.10.4")
+	testImplementation("org.testcontainers:testcontainers:1.20.0")
```

Comment:
I suggest to look closer at gradle dependencies catalog file
https://docs.gradle.org/current/userguide/platforms.html#sub::toml-dependencies-format

Reply:
do you mean we should store our dependencies somewhere else?

Reply:
Yes. I'll show you how to store deps.